src/actions/movieActions.js

```diff
@@ -0,0 +1,30 @@
+import Constants from '../components/Common/constants';
+import * as types from './actionTypes';
+
+export function fetchMovieDetailsSuccess(movie) {
+    return {
+        type: types.FETCH_MOVIE_DETAILS_SUCCESS,
+        movie
+    }
+}
+export function fetchSimilarMoviesSuccess(genre, response) {
+    return {
+        type: types.FETCH_SIMILAR_MOVIES_SUCCESS,
+        selectedGenre: genre,

```

Comment:
You can pass this two informations inside `payload` prop and the reducer that know the domain will know what to do with that. If you need to add or update any property will be an easier move and will be also included in FSA :D

Reply:
Okay